dialogue-serde/src/test/java/com/palantir/conjure/java/dialogue/serde/TracedEncodingTest.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.conjure.java.dialogue.serde;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+import com.google.common.io.ByteStreams;
+import com.palantir.dialogue.TypeMarker;
+import com.palantir.tracing.AlwaysSampler;
+import com.palantir.tracing.Tracer;
+import com.palantir.tracing.api.Span;
+import com.palantir.tracing.api.SpanObserver;
+import java.io.ByteArrayInputStream;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+
+class TracedEncodingTest {
+
+    @Test
+    void testTypeString_string() {
+        // non-generic types use the simple name
+        assertThat(TracedEncoding.toString(new TypeMarker<String>() {})).isEqualTo("String");
+    }
+
+    @Test
+    void testTypeString_stringList() {
+        // It's more complicated to generate 'List<String>' from Type, that can be done later if necessary
+        assertThat(TracedEncoding.toString(new TypeMarker<List<String>>() {}))
+                .isEqualTo("java.util.List<java.lang.String>");
+    }
+
+    @Test
+    void testSerializerOperationName() {
```

Comment:
these tests will not parallelize well :(

Reply:
luckily `./gradlew build` currently takes 1m22 so maybe 🤷

Reply:
didn't you configure everything to run in parallel by default?

Reply:
Nah i gave up because of our `@TestTracing` methods, which can't run in parallel with _anything_ else.

The only `junit-platform.properties` is in :simulation
```
junit.jupiter.execution.parallel.enabled=true
junit.jupiter.execution.parallel.config.strategy=dynamic
```